content/documentation/java-for-arm.md

```diff
@@ -68,7 +67,7 @@ Serial		: 000000005f9ba615
 Model		: Raspberry Pi Model B Plus Rev 1.2
 ```
 
-#### Install Java 11 on ARMv6
+### Install Java 11 on ARMv6
```

Comment:
Java 11? Isn't pi4j already on Java 17?

Reply:
Sorry, my bad, we are still on 11. This is bad. We should jump up to 21 asap. And i am looking into foreign function API which is only available on 22, this would make native code calls much simpler and less error prone.